WcaOnRails/app/views/static_pages/teams_committees.html.erb

```diff
@@ -13,7 +13,13 @@
     <% @officers.each do |user, positions| %>
       <div class="badge team-member-badge officer-badge">
         <%= render "shared/user_avatar", user: user %>
-        <div class="team-member-name"><%= user.name %>
+        <% if user.wca_id %>
+          <%= link_to person_path(user.wca_id), title: t("about.structure.users.profile", user_name: user.name) do %>
```

Comment:
I think we can take advantage of bootstrap's tooltips here (to display the same kind of tooltip that appear when you hover events [here](https://www.worldcubeassociation.org/results/rankings/333/single) for instance).
To do so, you should add `data: { toggle: "tooltip", placement: "bottom" }` to the `link_to` call.